src/Microsoft.ML.Data/StaticPipe/StaticSchemaShape.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.ML.StaticPipe.Runtime
     /// <summary>
     /// A schema shape with names corresponding to a type parameter in one of the typed variants
     /// of the data pipeline structures. Instances of this class tend to be bundled with the statically
-    /// typed variants of the dynamic structures (e.g., <see cref="DataView{TShape}"/> and so forth),
+    /// typed variants of the dynamic structures (for example, <see cref="DataView{TTupleShape}"/> and so forth),
```

Comment:
Was renaming `TShape` to `TTupleShape` intentional?

Reply:
Nope. I probably did a wrong analysis when resolving conflicts. The only changes should be spaces and e.g. -> for example